db/migrate/20230411031520_add_system_to_sidebar_sections.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddSystemToSidebarSections < ActiveRecord::Migration[7.0]
+  def change
+    add_column :sidebar_sections, :system, :boolean, default: false, null: false
```

Comment:
Keeping in mind that other sections like the categories, tags and messages section may also have to be moved into the database eventually, I'm wondering if a boolean column here is sufficient here.